pkg/render/intrusion_detection.go

```diff
@@ -292,7 +292,7 @@ func (c *intrusionDetectionComponent) intrusionDetectionElasticsearchJob() *batc
 		Spec: corev1.PodSpec{
 			Tolerations:      c.cfg.Installation.ControlPlaneTolerations,
 			NodeSelector:     c.cfg.Installation.ControlPlaneNodeSelector,
-			RestartPolicy:    corev1.RestartPolicyOnFailure,
+			RestartPolicy:    corev1.RestartPolicyNever,
```

Comment:
I think you should add a comment that this is set to RestartPolicyNever since the PodFailurePolicy will ensure it is ran until it passes.